src/lib.rs

```diff
@@ -1087,10 +1100,13 @@ mod tests {
         let stopped_actor = system.spawn(TestActor).unwrap().recipient();
 
         let error = full_actor.send(123).unwrap_err();
```

Comment:
It seems CI is failing on this line. I'm curious why it passed before. We're creating a zero-capacity channel, and flume's documentation has this to say:

> Like std::sync::mpsc, flume supports ‘rendezvous’ channels. A bounded queue with a maximum capacity of zero will block senders until a receiver is available to take the value. You can imagine a rendezvous channel as a [‘Glienicke Bridge’](https://en.wikipedia.org/wiki/Glienicke_Bridge)-style location at which senders and receivers perform a handshake and transfer ownership of a value.

https://docs.rs/flume/0.10.11/flume/fn.bounded.html

Reply:
This that was non-deterministic, failing in like 20% of the cases locally.

I didn't know why... until you've looked up that capacity zero is special, which I was not aware of! :) Thank you, I'll change the test to use non-zero capacity.

Reply:
Resolved in e814850.